src/main/java/com/metabug/service/TicketService.java

```diff
@@ -18,22 +21,18 @@ public TicketService(final TicketRepository ticketRepository,
         this.userService = userService;
     }
 
-    public Ticket findUserByEmail(final String title) {
-        return ticketRepository.findByTitle(title);
+    public List<Ticket> findAll() {
+        return ticketRepository.findAll();
     }
 
-    public void save(final TicketDto ticketDto) {
+    public void save(final TicketDto ticketDto, final Principal principal) {
         final Ticket ticket = new Ticket();
         ticket.setTitle(ticketDto.getTitle());
         ticket.setDescription(ticketDto.getDescription());
 
-        final String author = ticketDto.getAuthor();
-
-        if (userService.findUserByLogin(author) != null) {
-            ticket.setAuthorId(userService.findUserByLogin(author).getId());
-        } else {
-            ticket.setAuthorId(userService.findUserByEmail(author).getId());
-        }
+        //TODO: check if users exists? just in case?
+        ticket.setAuthorId(userService.findUserByLogin(principal.getName())
+                .getId());
```

Comment:
.getId one space to the left, so it aligns to the .setAuthorId

Reply:
Formatter, aligned to one-line